app/mailers/portal_mailer.rb

```diff
@@ -0,0 +1,13 @@
+class PortalMailer < ApplicationMailer
+
+  # @param hideRecipients [Boolean], recipients [Array<String>], reply_to [Array<String>], subject [String], content [String]
+  # @return [ActionMailer::MessageDelivery] a mail object with the given parameters.
+  def generic_email(hideRecipients, recipients, reply_to, subject, content)
```

Comment:
Thanks for the link :) I don't understand the reason nonetheless and for me it is even a violation of the uniform access principle. But probably there is a good reason for doing so :)